bluesky/run_engine.py

```diff
@@ -91,6 +91,8 @@ def __set__(self, obj, value):
      Parameter('subs', Parameter.POSITIONAL_ONLY, default=None),
      Parameter('metadata_kw', Parameter.VAR_KEYWORD)])
 
+def default_scan_id_source(md):
```

Comment:
To keep style-checkers happy, please leave _two_ blank lines above and below top-level functions like this one.